controllers/snapshot/snapshot_adapter.go

```diff
@@ -109,6 +108,11 @@ func (a *Adapter) EnsureAllIntegrationTestPipelinesExist() (reconciler.Operation
 					a.logger.Error(err, "Failed to create pipelineRun for snapshot and scenario")
 					return reconciler.RequeueOnErrorOrStop(err)
 				}
+				a.logger.Info("IntegrationTestscenario pipeline has been created",
```

Comment:
Should we also add a `a.logger.LogAuditEvent` call with `h.LogActionAdd` action here?

Reply:
yes, but I cannot find reference to created pipeline, it may need more code changes

Reply:
normally OCP returns object in POST response, but with this k8s runtime client, I don't know how to get name of object when name is generated dynamically

Reply:
I was blind :| I'll do it in separate commit

Reply:
added

Reply:
Brilliant, looks good